src/23-2017-q1-update/index.md

```diff
@@ -0,0 +1,62 @@
+---
+# this is the final blog post's id (used in the directory)
+# this id is part of the url, and should only contain:
+#   letters, numbers, dashes.
+id: 2017-q1-update
+
+breadcrumbs:
+  - {name: "2017-q1-update", link: "./" }
+
+# the date here should be set to the final publication date,
+# on the day it is published.
+date: 2017-01-25
+
+# this is the Title
+title: IPFS Project Management Update for Q1 2017
+
+# this is the name of the main author(s)
+author: Richard Littauer
+
+# technical details required for the software, don't change these.
+baseurl: ..
+template: tmpl/layouts/post.html
+collection: posts
+---
+
+# IPFS going into Q1 in 2017
+
+This year, IPFS is changing the way we do our sprints. This post covers some of the basic aspects of these changes. A longer post will be coming in February.
```

Comment:
The IPFS team are trying out a new format for arranging our work and our roadmap. This post covers some of the basic aspects of these changes. A longer post will be coming in February.